src/main/java/today/sesac/shoutify/post/dto/response/PostSingleQueryResponseDto.java

```diff
@@ -0,0 +1,37 @@
+package today.sesac.shoutify.post.dto.response;
+
+import java.time.LocalDateTime;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+/**
+ * 게시글 단건 조회 응답 dto.
+ */
+@Getter
+@AllArgsConstructor(staticName = "of")
+public class PostSingleQueryResponseDto {
+
+    private final Long postId;
+
+    private final String title;
+
+    private final String afterContent;
+
+    private final String nickname;
+
+    private final LocalDateTime createdAt;
+
+    private final String imgUrl;
+
+    private final String conceptType;
+
+    private final boolean isMine;
```

Comment:
P2
이거 혹시 확인해 보셨나요? 제가 확인했을 때는 mine으로 응답이 옵니다.

jackson 라이브러리에서는 boolean 변수명에서 is를 떼고 json으로 변환해 줍니다. 그래서 is를 살리고 싶다면, 간단하게는 Boolean으로 하면 postman에서 isMine으로 보입니다. 그 외의 방법은 참고를 확인해 주세요.

```suggestion
    private final Boolean isMine;
```

참고: [[JAVA] Jackson사용 시 primitive boolean주의점](https://00h0.tistory.com/93)

Reply:
진짜 몰랐던 사실..오오오오오 대박 감사합니다!
[49d1075](https://github.com/SproutJavaSesac/shoutify-backend/pull/60/commits/49d1075b0e9392391aadfc79d9a8679ddf354942)
[f0c9afc](https://github.com/SproutJavaSesac/shoutify-backend/pull/60/commits/f0c9afcbe924c6e3e820071a31674a76add41668)